convert2rhel/breadcrumbs.py

```diff
@@ -0,0 +1,191 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright(C) 2016 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import json
+import os
+import sys
+
+from time import gmtime, strftime
+
+from convert2rhel import pkghandler, systeminfo
+
+
+# is importing systeminfo good idea? It creates new instance of SystemInfo after import
+# but we aren't setting the data, so might be ok
+
+FILE = "/etc/migration-results"
+
+
+class Breadcrumbs(object):
+    def __init__(self):
+        self.executed = ""
+        self.nevra = None
+        self.signature = None
+        self.exit_code = None
+        self.timestamp = None
```

Comment:
```suggestion
        self.activity_started = None
        self.activity_ended = None
```
@shaded-enmity can help with implementing/getting the `activity_started`.